docs/troubleshooting/management/ts-management.asciidoc

```diff
@@ -12,34 +12,34 @@ This topic covers common troubleshooting issues when using {elastic-sec} <<sec-m
 .Unhealthy {agent} status
 [%collapsible]
 ====
-In some cases, an `Unhealthy` {agent} status may be caused by a failure in the {endpoint-cloud-sec} integration policy. In this situation, the integration and any failing features are flagged on the agent details page in {fleet}. Expand each section and subsection to display individual responses from the agent.
+In some cases, an `Unhealthy` {agent} status may be caused by a failure in the {elastic-defend} integration policy. In this situation, the integration and any failing features are flagged on the agent details page in {fleet}. Expand each section and subsection to display individual responses from the agent.
 
 TIP: Integration policy response information is also available from the *Endpoints* page in the {security-app} (*Manage* -> *Endpoints*, then click the link in the *Policy status* column).
 
 [role="screenshot"]
```

Comment:
This image still shows the old cloud integration in the agent policy. Feel free to ignore this comment if you're already tracking this :)

Reply:
Screenshot updated!